src/tests/stub_MemStore.cc

```diff
@@ -36,6 +36,7 @@ uint64_t MemStore::currentCount() const STUB_RETVAL(0)
 int64_t MemStore::maxObjectSize() const STUB_RETVAL(0)
 bool MemStore::dereference(StoreEntry &) STUB_RETVAL(false)
 void MemStore::markForUnlink(StoreEntry&) STUB
+void MemStore::unlinkByKeyIfFound(const cache_key *key) STUB
```

Comment:
please remove the parameter name here. it leads to compile error 'unused variable' on strict compilers.

same in new stub_store.cc and TestSwapDir.h lines.